### PR TITLE
MGDOBR-971: Verify new Azure Event Hubs source schema

### PR DESCRIPTION
**Pull request by kelvah**
JIRA: https://issues.redhat.com/browse/MGDOBR-971

### CODE REVIEW

**Review by vpellegrino — APPROVED** [review]
LGTM

**Review by manstis — APPROVED** [review]
LGTM :+1:

File: mocked-api/schemasData.ts [hunk 1]
```diff
@@ -785,4 +793,80 @@ export const schemasData: { [key: string]: object } = {
...
+      azure_eventhub_name: {
+        title: "Eventhubs Name",
+        description: "The Event Hub name.",
+        type: "string",
+      },
+      azure_shared_access_name: {
+        title: "Share Access Name",
+        description: "The Event Hubs SAS key name.",
+        type: "string",
+        "x-group": "credentials",
```
Comment (sjamboro) [inline mocked-api/schemasData.ts:823]: @kelvah please can you share a definition about "x-group:gredentials" I thought that it marks that the parameter should be hidden but the value is still visible. See: 
image

File: mocked-api/schemasData.ts [hunk 2]
```diff
@@ -785,4 +793,80 @@ export const schemasData: { [key: string]: object } = {
...
+      },
+      azure_shared_access_key: {
+        title: "Share Access Key",
+        "x-group": "credentials",
+        oneOf: [
+          {
+            title: "Share Access Key",
+            description: "The key for the EventHubs SAS key name.",
+            type: "string",
+            format: "password",
```
Comment (sjamboro) [inline mocked-api/schemasData.ts:833]: @kelvah probably this is not a problem of this issue but the value of this property is visible in Process Details. Is it a know issue?
image

**Review by sjamboro — APPROVED** [review]
thanks @kelvah ... I have left minor comments.

**Comment by sjamboro** [comment]
@kelvah there will be problem with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/116. Do you want to merge this first?

File: mocked-api/schemasData.ts (hunk 1 above)
Comment (kelvah) [reply mocked-api/schemasData.ts:823]: What makes a parameter hidden is its actual definition. 
It has to include the `oneOf` property with the double field definition: one with `format="password"` and the other with `type="object"`
You can see an example here:
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/8d96ac75b463442d90c9cd277935847271d1d3e5/mocked-api/schemasData.ts#L205-L221

I'm not sure what the `x-group` property stands for. It's clearly not a json-schema prop. We inherit these from the original kamelet used to implement the connector. See [here for example](https://github.com/apache/camel-kamelets/blob/main/kamelets/azure-eventhubs-source.kamelet.yaml#L53-L67). But we are not actually using them.

File: mocked-api/schemasData.ts (hunk 2 above)
Comment (kelvah) [reply mocked-api/schemasData.ts:833]: This will be covered by #101.

**Comment by kelvah** [comment]
> @kelvah there will be problem with #116. Do you want to merge this first?

OK. Thanks.

**Pull request merged**